.changeset/giant-shoes-reflect.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/cli': minor
```

Comment:
```suggestion
'@backstage/cli': patch
```

Reply:
My original thought was to keep it quite verbose on local development but silence it through the flag. e.g. notify developer about potential issues but keep CI logs silent. But I'll check the current verbosity level and what these change.